unpackaged/config/qa/objects/Lead.object

```diff
@@ -0,0 +1,1020 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It looks like we are only intending to deploy a single field on Lead. Can we excise the remainder of the metadata other than the handful of required attributes? I see some Data.com field references here that will be a problem when that feature ultimately goes away.